tests/migrations/test_questioner.py

```diff
@@ -0,0 +1,15 @@
+from __future__ import unicode_literals
+
+from django.db.migrations.questioner import MigrationQuestioner
+from django.test import SimpleTestCase
+from django.test.utils import override_settings
+
+
+class QuestionerTests(SimpleTestCase):
+    @override_settings(
+        INSTALLED_APPS=['migrations'],
+        MIGRATION_MODULES={'migrations': None},
+    )
+    def test_ask_initial_with_disabled_migrations(self):
+        questioner = MigrationQuestioner()
+        self.assertFalse(questioner.ask_initial('migrations'))
```

Comment:
I often wonder whether it's a good idea to use `assertFalse` in cases like this since it passes for anything "Falsey" like `None`.